content/blog/kubernetes-yaml-generation/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Generate Kubernetes YAML with Real Programming Languages"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2020-02-21T13:11:29-07:00
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting your post for review.
+draft: false
+
+# Use the optional meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. If omitted or left blank, the content preceding the `<!--more-->` token
+# will be used in its place.
+meta_desc: Stop writing Kubernetes YAML by hand, and start using the power of real programming languages! Pulumi now supports rendering YAML for Kubernetes resources.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: yamlRuler.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - levi-blackstone
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - kubernetes
+    - yaml
+    - kubernetesx
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Stop writing Kubernetes YAML by hand, and start using the power of real programming languages! Pulumi now allows you to
+generate Kubernetes manifests that can be easily integrated into existing CI/CD workflows.
+
+<!--more-->
+
+While Pulumi has great support for deploying and updating Kubernetes resources on a cluster, many users have asked for
+the option to render YAML that they can integrate into existing workflows. The `v1.5.4` release of `pulumi-kubernetes`
+adds the `renderYamlToDirectory` option, which enables this behavior.
+
+This option is available in every Pulumi-supported language, including TypeScript/JavaScript, Python, and .NET
+(Go support is coming soon!). Aside from easily templating configuration across resources, using a real programming
```

Comment:
Would be good to link to the go issue in case anyone wants to follow along: https://github.com/pulumi/pulumi-kubernetes/issues/70